apps/wing/src/commands/run.test.ts

```diff
@@ -85,3 +111,43 @@ test("wing it with an invalid file throws exception", async () => {
     process.chdir(prevdir);
   }
 });
+
+test("wing it with a custom port runs", async () => {
+  const workdir = await mkdtemp(join(tmpdir(), "-wing-it-test"));
+  const prevdir = process.cwd();
+  try {
+    process.chdir(workdir);
+
+    writeFileSync("foo.w", "bring cloud;");
+
+    await run("foo.w", { port: "5000" });
+    expect(createConsoleApp).toBeCalledWith({
+      wingfile: resolve("foo.w"),
+      requestedPort: 5000,
+      hostUtils: expect.anything(),
+    });
+    expect(open).toBeCalledWith("http://localhost:5000/");
+  } finally {
+    process.chdir(prevdir);
+  }
+});
+
+test("wing it with an invalid port ignores it", async () => {
```

Comment:
Thanks for adding the test - makes it easier to spot that this is what happens when an invalid `port` is passed.
It is better to show an error to the user that they passed wrong input than silently ignore.